src/Aspire.Dashboard/Configuration/ValidateDashboardOptions.cs

```diff
@@ -17,14 +17,23 @@ public ValidateOptionsResult Validate(string? name, DashboardOptions options)
             errorMessages.Add(frontendParseErrorMessage);
         }
 
-        if (!options.Otlp.TryParseOptions(out var otlpParseErrorMessage))
+        switch (options.Frontend.AuthMode)
         {
-            errorMessages.Add(otlpParseErrorMessage);
+            case FrontendAuthMode.Unsecured:
+                break;
+            case FrontendAuthMode.OpenIdConnect:
+                break;
+            case null:
+                errorMessages.Add($"Frontend endpoint authentication is not configured. Either specify {DashboardConfigNames.DashboardUnsecuredAllowAnonymousName.ConfigKey}=true, or specify {DashboardConfigNames.DashboardFrontendAuthModeName.ConfigKey}. Possible values: {string.Join(", ", typeof(FrontendAuthMode).GetEnumNames())}");
```

Comment:
Error messages should be localised.

Reply:
If we want to do that then create a new issue. It's beyond the scope of this PR.